### PR TITLE
188 support tablet screen size

### PR DESCRIPTION
**Pull request by zpoken**

### CODE REVIEW

File: apps/webapp/app/dashboard/assets-table.tsx [hunk 1]
```diff
@@ -56,19 +56,21 @@ export default function AssetsTable() {
                   <TableRow key={i}>
                     <TableCell className='w-1/3'>
                       <div className='flex flex-col items-center gap-4 '>
-                        <p className=' font-mono text-base'>{asset.denom.display}</p>
+                        <p className='font-mono font-bold md:text-[12px] md:leading-[18px] xl:text-base'>
```
[the remark this answers is not in the document]
Comment (grod220) [reply apps/webapp/app/dashboard/assets-table.tsx:59]: This text should be back at its normal size: 

Screenshot 2023-11-07 at 2 59 48 PM

File: apps/webapp/app/dashboard/layout.tsx
Comment (grod220) [inline apps/webapp/app/dashboard/layout.tsx:1]: In tablet view this gets larger? Think you should keep it as is.

Screenshot 2023-11-07 at 3 00 40 PM

File: apps/webapp/app/dashboard/transactions/transaction-table.tsx
Comment (grod220) [inline apps/webapp/app/dashboard/transactions/transaction-table.tsx:1]: Font way to small. Please revert. Anywhere you've made the smallest text size smaller, revert.

File: apps/webapp/app/header/header.tsx
Comment (grod220) [inline apps/webapp/app/header/header.tsx:1]: When on transactions page (`/tx?hash=941a1d9d4dfe9142d2e5`), its empty:

Screenshot 2023-11-07 at 3 03 06 PM

File: apps/webapp/app/header/tablet-sidebar.tsx
Comment (grod220) [inline apps/webapp/app/header/tablet-sidebar.tsx:1]: It should not be called "sidebar" if it's on top. I'd call it `tablet-nav-menu`

File: apps/webapp/app/header/tablet-sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  NavigationMenuTrigger,
+  navigationMenuTriggerStyle,
+} from '@penumbra-zone/ui';
+import Link from 'next/link';
+import { headerLinks } from './constants';
+import { cn } from '@penumbra-zone/ui/lib/utils';
+import { useTypedPathname } from '../../hooks/typed-pathname';
+import { DappPath } from '../../shared/header/types';
+
+export const TabletSidebar = () => {
```
Comment (grod220) [inline apps/webapp/app/header/tablet-sidebar.tsx:17]: When it's first open, it's empty for a half second. Please fix.

Screenshot 2023-11-07 at 3 05 49 PM

File: apps/webapp/shared/edu-panels/content.ts [hunk 3]
```diff
@@ -19,5 +19,5 @@ export const eduPanelContent: Record<EduPanel, string> = {
...
     'Every Penumbra account has a stable default address and many one-time addresses.  All addresses for the same account deposit to the same pool of funds.  Use a freshly generated one-time address in situations where you need to reveal an address publicly, like an IBC transfer from a transparent chain into Penumbra.',
   [EduPanel.TEMP_FILLER]:
-    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
```
Comment (grod220) [inline apps/webapp/shared/edu-panels/content.ts:22]: Why remove half of this?

File: apps/webapp/app/send/ibc/ibc-form.tsx [hunk 4]
```diff
@@ -83,7 +83,7 @@ export default function IbcForm() {
           </SelectContent>
         </Select>
       </div>
-      <Button type='submit' variant='gradient' className='mt-9' disabled={true}>
+      <Button type='submit' variant='gradient' className='md:mt-6 xl:mt-2' disabled={true}>
```
Comment (grod220) [inline apps/webapp/app/send/ibc/ibc-form.tsx:86]: Asset text is way too small, can barely read it

File: apps/webapp/app/send/send-form.tsx
Comment (grod220) [inline apps/webapp/app/send/send-form.tsx:1]: This error does not really work at the tablet view. Please work with Daria to come up with something else.

Screenshot 2023-11-07 at 3 10 54 PM

File: packages/ui/components/ui/popover.tsx [hunk 5]
```diff
@@ -18,7 +18,7 @@ const PopoverContent = React.forwardRef<
       align={align}
       sideOffset={sideOffset}
       className={cn(
-        'z-50 w-[240px] rounded-lg bg-background p-4 text-muted-foreground shadow-md outline-none data-[state=open]:animate-in data-[state=open]:fade-in-0 data-[state=open]:zoom-in-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[state=closed]:fill-mode-forwards data-[state=open]:fill-mode-forwards',
+        'z-50 w-[240px] rounded-lg bg-red-900 p-4 text-muted-foreground shadow-md outline-none data-[state=open]:animate-in data-[state=open]:fade-in-0 data-[state=open]:zoom-in-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[state=closed]:fill-mode-forwards data-[state=open]:fill-mode-forwards',
```
Comment (grod220) [inline packages/ui/components/ui/popover.tsx:21]: Did the color change? Can you give a screenshot?

File: packages/ui/components/ui/switch.tsx
Comment (grod220) [inline packages/ui/components/ui/switch.tsx:1]: It looks like extra margin is put below the switch now? The margin is not the same as large screen size.

Screenshot 2023-11-07 at 3 14 55 PM

File: apps/webapp/app/header/header.tsx
Comment (zpoken) [reply apps/webapp/app/header/header.tsx:1]: there should be Dashboard? maybe make sense  - /dashboard/transactions/941a1d9d4dfe9142d2e5

File: apps/webapp/shared/edu-panels/content.ts (hunk 3 above)
Comment (zpoken) [reply apps/webapp/shared/edu-panels/content.ts:22]: just looked at what swap would look like with smaller education blocks

Comment (zpoken) [reply apps/webapp/shared/edu-panels/content.ts:22]: I can return it, but it's a test text

File: apps/webapp/app/header/header.tsx
Comment (grod220) [reply apps/webapp/app/header/header.tsx:1]: If it says Dashboard, then users would not be able to navigate back to the Dashboard. It should probably say Transaction.

File: packages/ui/components/ui/switch.tsx
Comment (zpoken) [reply packages/ui/components/ui/switch.tsx:1]: what do you mean?
i do like there- https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=3188%3A1720&mode=dev

Comment (grod220) [reply packages/ui/components/ui/switch.tsx:1]: Look at the difference between tablet and desktop for that component. For tablet, more margin is added. I'm saying that we should not add more margin just for tablets.

File: apps/webapp/app/send/send-form.tsx
Comment (zpoken) [reply apps/webapp/app/send/send-form.tsx:1]: Знімок екрана 2023-11-08 о 11 08 37

is it ok? for 768px screen

File: packages/ui/components/ui/switch.tsx
Comment (zpoken) [reply packages/ui/components/ui/switch.tsx:1]: Знімок екрана 2023-11-08 о 11 16 07

add same margin as desktop

File: apps/webapp/app/dashboard/layout.tsx
Comment (grod220) [reply apps/webapp/app/dashboard/layout.tsx:1]: It's still growing it seems

Screenshot 2023-11-08 at 10 30 13 AM

File: apps/webapp/app/send/send-form.tsx
Comment (grod220) [reply apps/webapp/app/send/send-form.tsx:1]: The problem is that it makes the input jump down. We need a design that doesn't resize the box and make it jump

File: packages/ui/components/ui/switch.tsx
Comment (grod220) [reply packages/ui/components/ui/switch.tsx:1]: seems like this was fixed on the last changes

**Review by grod220 — APPROVED** [review]
Fix those last things and you can merge

File: apps/webapp/app/dashboard/layout.tsx
Comment (zpoken) [reply apps/webapp/app/dashboard/layout.tsx:1]: only for screens 768px-1023px! it small screen

Comment (grod220) [reply apps/webapp/app/dashboard/layout.tsx:1]: 768px-1023px is tablet size though

Comment (grod220) [reply apps/webapp/app/dashboard/layout.tsx:1]: Reference:

```

Mobile devices (Small): This usually includes devices with screen widths less than 768px.

Tablets (Medium): This typically includes devices with screen widths from 768px to 1023px.

Desktops/Laptops (Large): This generally includes devices with screen widths greater than 1024px.
```

Comment (zpoken) [reply apps/webapp/app/dashboard/layout.tsx:1]: Знімок екрана 2023-11-08 о 15 10 03

how to look for 768px! do I need to decrease the tab here?

Comment (grod220) [reply apps/webapp/app/dashboard/layout.tsx:1]: I'd say when it's this small, it should go to the mobile layout (one column)

**Pull request merged**